documentation/examples/prometheus-kubernetes.yml

```diff
@@ -0,0 +1,33 @@
+# A scrape configuration containing exactly one endpoint to scrape:
+# Here it's Prometheus itself.
+global:
+  scrape_interval:     15s # By default, scrape targets every 15 seconds.
+  evaluation_interval: 15s # By default, scrape targets every 15 seconds.
+  scrape_timeout: 10s
+
+scrape_configs:
+# The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+- job_name: 'prometheus'
+
+  target_groups:
+  - targets: ['localhost:9090']
+
+- job_name: 'kubernetes'
+
+  kubernetes_sd_configs:
+  - server: 'https://kubernetes.default.svc'
+    in_cluster: true
+
+  metric_relabel_configs:
```

Comment:
I guess it is tbh. Let me see if I can get a PR into kubernetes for that.
Otherwise this is going to be pretty heavy on Prometheus.

On 13:34, Wed, 12 Aug 2015 Brian Brazil <notifications@github.com> wrote:

> In documentation/examples/prometheus-kubernetes.yml
> <https://github.com/prometheus/prometheus/pull/905#discussion_r36853872>:
>
> > +  scrape_timeout: 10s
> > +
> > +scrape_configs:
> > +# The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
> > +- job_name: 'prometheus'
> > +
> > +  target_groups:
> > +  - targets: ['localhost:9090']
> > +
> > +- job_name: 'kubernetes'
> > +
> > +  kubernetes_sd_configs:
> > +  - server: 'https://kubernetes.default.svc'
> > +    in_cluster: true
> > +
> > +  metric_relabel_configs:
>
> This is going to affect all scraped kubernetes metrics, so you'll need to
> be much more specific on the source labels to only affect the right target.
> In fact given that this is a regex applied to every ingested sample, you
> may want a separate scrape config for it.
>
> I don't suppose this is something the kublet could do itself?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/prometheus/prometheus/pull/905/files#r36853872>.
>